submissions/ZAIRIyoussef.py

```diff
@@ -0,0 +1,44 @@
+import csv
+from collections import defaultdict
```

Comment:
are these imports from standard python libraries ?

Reply:
@aboullaite, I'm answering on Youssef's behalf, yes, both are part of the standard Python library